src/libraries/System.Diagnostics.Process/tests/ProcessStartInfoTests.cs

```diff
@@ -1347,5 +1317,200 @@ private void VerifyNotepadMainWindowTitle(Process process, string filename)
 
             Assert.StartsWith(expected, title);
         }
+
+        [ConditionalFact(typeof(PlatformDetection), nameof(PlatformDetection.IsNotWindowsNanoServer))] // No Notepad on Nano
+        [OuterLoop("Launches notepad")]
+        [PlatformSpecific(TestPlatforms.Windows)]
+        public void StartInfo_LoadUserProfile_And_UseCredentialsForNetworkingOnly_AreIncompatible()
+        {
+            ProcessStartInfo info = new ProcessStartInfo
+            {
+                LoadUserProfile = true,
+                UseCredentialsForNetworkingOnly = true,
+                UserName = "dummy",
+                PasswordInClearText = "not used, because ArgumentException should be thrown before",
+                FileName = "notepad.exe",
+                Arguments = null,
+                WindowStyle = ProcessWindowStyle.Minimized
+            };
+
+            Assert.Throws<ArgumentException>("startInfo", () =>
+            {
+                using (var process = Process.Start(info))
+                {
+                    Assert.False(process != null, $"Process started despite incompatible options {nameof(info.LoadUserProfile)} and {nameof(info.UseCredentialsForNetworkingOnly)} were enabled");
+                }
+            });
+        }
+
+        [ConditionalFact(nameof(IsAdmin_IsNotNano_RemoteExecutorIsSupported))] // Nano has no "netapi32.dll", Admin rights are required
```

Comment:
Sounds reasonable. Will update the `TestFileShare` class accordingly.

Reply:
Due to the dependency of this method on `ServiceController`, this would require the `TestUtilities` project to take a dependency on the `System.ServiceProcess.ServiceController` library.

What's the *correct* way to do so for the `TestUtilities` project? The `System.IO.FileSystem.Tests` currently uses the `csproj`
```xml
<ProjectReference Include="$(LibrariesProjectRoot)System.ServiceProcess.ServiceController\src\System.ServiceProcess.ServiceController.csproj" />
```
but the `TestUtilities` don't reference library projects in that manner, yet. Using it like this would make the proj compile for the `net6.0` target, but not for the `net462` target. Also, would that require every other `sln` that uses the `TestUtilities` to also reference the `System.ServiceProcess.ServiceController.csproj`?

Reply:
Sounds good to me!